docs/guide.md

```diff
@@ -176,7 +173,7 @@ res15: Int = 2
 
 Here we finally see the tasks being executed. As is shown with `rc`, rerunning a task executes the entire computation again; nothing is cached for you automatically.
 
-_Note:_ The various `run*` functions aren't specialized to `Task` and work for any `F[_]` with an implicit `Catchable[F]`---FS2 needs to know how to catch errors that occur during evaluation of `F` effects.
+_Note:_ The various `run*` functions aren't specialized to `Task` and work for any `F[_]` with an implicit `MonadError[F, Throwable]`---FS2 needs to know how to catch errors that occur during evaluation of `F` effects.
```

Comment:
IMO no, as I generally feel that type aliases add to the cognitive load needed to understand a library. They create a new name for an existing concept, requiring folks to know both names when puzzling through error messages. I'm fine with the type alias existing if others want it though.